modules/compliance-timeout.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-timeout_{context}"]
+= Configuring ScanSetting timeout
+
+The `ScanSetting` object has a timeout option that can be specified in the `ComplianceScanSetting` object as a duration string, such as 1h30m. If the scan does not finish within the specified timeout, the scan will be reattempted until the `maxRetryOnTimeout` limit is reached.
```

Comment:
nit: should 1h30m be a literal?

Reply:
Probably should be in backticks, yeah - `1h30`. I'll make that change!